src/connectedk8s/azext_connectedk8s/_params.py

```diff
@@ -51,6 +51,11 @@ def load_arguments(self, _):
         c.argument('container_log_path', help='Override the default container log path to enable fluent-bit logging')
         c.argument('skip_ssl_verification', action='store_true', help='Skip SSL verification for any cluster connection.')
         c.argument('yes', options_list=['--yes', '-y'], help='Do not prompt for confirmation.', action='store_true')
+        c.argument('enable_gateway', options_list=['--enable_gateway'], help='pass this value to enable Arc Gateway')
```

Comment:
remove enable and disable gateway as these params are into part of your pr and we will add logic and extra validations later